mars/scheduler/operands/base.py

```diff
@@ -79,7 +79,7 @@ def post_create(self):
         self.set_cluster_info_ref()
         self._assigner_ref = self.ctx.actor_ref(AssignerActor.default_name())
         self._chunk_meta_ref = self.ctx.actor_ref(ChunkMetaActor.default_name())
-        self._graph_ref = self.get_actor_ref(GraphActor.gen_name(self._session_id, self._graph_id))
+        self._graph_refs.append(self.get_actor_ref(GraphActor.gen_name(self._session_id, self._graph_ids[0])))
```

Comment:
Why not combine this into line 65 and use graph instead of self._graph_ids[0]?

Reply:
obtaining actor ref should be in `post_create`